src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -121,6 +127,21 @@ public void cancelTest() {
         awaitCondition(() -> toggle.get() % 2 == 1);
     }
 
+    @Test
+    public void testHelp() {
```

Comment:
Understood, I will favor the official coding standards over file-local conventions.